index.js

```diff
@@ -1,20 +1,258 @@
+    heading: 'Keeping track of hundreds of components',
+    technologies: ['Ruby on Rails', 'CSS', 'JavaScript', 'Bootstrap'],
+    paragraph:
+    'Lorem Ipsum is simply dummy text of the printing and typesetting industry Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. <br/> <br/> Lorem Ipsum is simply dummy text of the printing and typesettin industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.',
+    button1: 'https://github.com/Mike47ip/mobilePortfolioWeb',
+    button2: 'https://github.com/Mike47ip/mobilePortfolioWeb',
+  },
+  {
+    id: '6',
+    image: '/Images/Snapshoot-Portfolio.png',
+    alt: 'Popup Window Image',
+    heading: 'Keeping track of hundreds of components',
+    technologies: ['Ruby on Rails', 'CSS', 'JavaScript', 'Bootstrap'],
+    paragraph:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. <br/> <br/> Lorem Ipsum is simply dummy text of the printing and typesettin industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.',
+    button1: 'https://github.com/Mike47ip/mobilePortfolioWeb',
+    button2: 'https://github.com/Mike47ip/mobilePortfolioWeb',
+  },
+];
+
+const openpop = document.querySelectorAll('.button1');
+const secpop = document.querySelector('.Secpopupwin');
+const closepop = document.querySelector('.Closepop');
+const popupimage = document.querySelector('.popupWindowImage');
+const popuppara = document.querySelector('.headingpop');
+const popupServer = document.querySelector('#serverWin');
+const popupLorem = document.querySelector('.Lorem');
+const live = document.querySelector('#live');
+const source = document.querySelector('#source');
+
+const openpopwin = () => {
+  secpop.classList.add('active');
+};
+
+const closepopwin = () => {
+  secpop.classList.remove('active');
+};
+
+closepop.addEventListener('click', closepopwin);
+
+openpop.forEach((windowpop) => {
+  windowpop.addEventListener('click', () => {
+    openpopwin();
+
+    projectPopupWindow.forEach((windowpopup) => {
+      if (windowpopup.id === windowpop.id) {
+        popupimage.src = windowpopup.image;
+        popuppara.textContent = windowpopup.heading;
+        popupServer.textContent = '';
+        windowpopup.technologies.forEach((tech) => {
+          const techlist = document.createElement('li');
+          techlist.innerText = tech;
+          popupServer.appendChild(techlist);
+        });
+        popupLorem.innerHTML = windowpopup.paragraph;
+        live.href = windowpopup.button1;
+        source.href = windowpopup.button2;
+      }
+    });
```

Comment:
- [ ] KIndly make sure to style the buttons as much as close to the design, spacings, margins, paddings, and icons, kindly address this issue.

Mobile view

| Your design | Actual design |
| ------------- | ------------- |
| ![Screenshot 2023-07-01 002159](https://github.com/Mike47ip/mobilePortfolioWeb/assets/102612396/d20f9758-845f-4280-bcf7-5d5fafb74bcf)  | ![Screenshot 2023-07-01 002210](https://github.com/Mike47ip/mobilePortfolioWeb/assets/102612396/eff044be-abc5-4d0b-b402-3db4fc8ea5d7) |

Desktop 

| Your design | Actual design |
| ------------- | ------------- |
| ![Screenshot 2023-07-01 002236](https://github.com/Mike47ip/mobilePortfolioWeb/assets/102612396/99c7443c-555b-4c69-883c-77b3aa2595fb) | ![Screenshot 2023-07-01 002243](https://github.com/Mike47ip/mobilePortfolioWeb/assets/102612396/90e07758-97fa-4e5a-af91-1b2bce442812) |